### ISSUE TITLE
Align box in center in mobile view

### ISSUE THREAD

**Issue by Paarmita** [publiclab/image-sequencer#484]
**We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

Hi, this is a first-timers-only issue. This means we've worked to make it more legible to folks who either haven't contributed to our codebase before or even folks who haven't contributed to open source before.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general help wanted issues. Thanks!

## The Problem:
We want the Download PNG and View GIF buttons div in the center in the mobile view.

## How to Solve?
You can add responsiveness in the div.

This line can be added below this one.
https://github.com/publiclab/image-sequencer/blob/bf13d5807ab34b04517a3c9d3b4c318342432a48/examples/index.html#L94

Instance Count 1

![screenshot from 2018-11-20 13-11-29](https://user-images.githubusercontent.com/26546816/48758592-91499b00-ecc6-11e8-9dba-f48fe80c0ea1.png)

**Comment by Rishabh570** [comment]
For anybody who would work on it, only removing [this](https://github.com/publiclab/image-sequencer/blob/bf13d5807a/examples/demo.css#L130) would made it sit in the center :)

**Comment by lwahlnayar** [comment]
Can I give it a go at this? I'm completely new to contributions =)

**Comment by Rishabh570** [comment]
@lwahlnayar Are you participating in GCI?

**Comment by lwahlnayar** [comment]
not really...

**Comment by Rishabh570** [comment]
Actually, this task is for GCI participants...I appreciate that you want to contribute, you can take a look at other open first-timer-only issues here: https://code.publiclab.org/

**Comment by lwahlnayar** [comment]
alright, thanks!

**Comment by JonathanXu1** [comment]
Hey there, could I work on this for GCI? This task doesn't seem to be available on the dashboard but I'm also not aware if someone else is working on it...

**Comment by Rishabh570** [comment]
Yes, you can work on it as it's not claimed by anybody yet...I've created a draft for it, pinging @jywarren for reviewing it and publishing it :)

**Comment by jywarren** [comment]
published!

**Comment by JonathanXu1** [comment]
Is there a link?

**Comment by JonathanXu1** [comment]
Oh it seems like this is a beginner task. My bad.

**Comment by AmanGarg97** [comment]
Hey! Can I work on this? It's my first contribution on image-sequencer.

**Comment by HarshKhandeparkar** [comment]
@AmanGarg97, sure! That will be awesome.

**Comment by VibhorCodecianGupta** [comment]
@AmanGarg97 I think you can take this one up since you are contributing to Image Sequencer for the first time, familiarise yourself with the code base.
@HarshKhandeparkar I think this is a good `fto` issue, let's let first-timers work on this, cheers!

**Comment by AmanGarg97** [comment]
@VibhorCodecianGupta @HarshKhandeparkar I can start with this but it would also be best if you can assign some another issue so that I can go through the codebase and can start contributing to this.
Thanks!

**Comment by AmanGarg97** [comment]
Hey please review #669 and let me know for any other changes.

**Comment by AmanGarg97** [comment]
Also are there any proper Contributions Guidelines for the project?
What I faced was:
1. How to install the project in Linux environment for a newbie.
2.What are the steps to solve an issue and make a PR (branching and all that stuff).

If there is not then I will love to make one.
Thanks!

**Comment by tech4GT** [comment]
I think we have a setup script Aman, try running `npm run setup` and then `npm start`

**Comment by HarshKhandeparkar** [comment]
@jywarren @gauravano @PaarmitaI think this is not required due to recent changes

![screenshot_20190127-110553](https://user-images.githubusercontent.com/34770591/51797106-d8529000-2223-11e9-9840-8760101aad05.jpg)


But there is a PR open for it https://github.com/publiclab/image-sequencer/pull/669
